src/Nethermind/Nethermind.Core.Test/MeasuredProgressTests.cs

```diff
@@ -84,7 +84,7 @@ public void Update_twice_total_per_second()
 
         [Test]
         [Retry(3)]
-        public async Task Update_twice_current_per_second()
+        public void Update_twice_current_per_second()
```

Comment:
This silences warnings about async functions without an await.  I saw some other tests that were void returning instead of `async Task` returning, so I think that is acceptable from a style standpoint.

Reply:
noooooo! :) this is anti pattern
use await Task.Completedl instead

Reply:
Out of curiosity, why do you consider `void` returning functions to be an anti-pattern?

Also, there are other tests in this file that are void returning which you may want to fix.  😄